lib/pure/asynchttpserver.nim

```diff
@@ -271,21 +271,8 @@ proc processRequest(
         await client.sendStatus("417 Expectation Failed")
 
```

Comment:
add `fix #17456` to PR message so that the issue will auto-close when this PR is merged

Reply:
Not sure which PR message you're referring to - I see that you changed the title of this PR; I also added it to the last commit message.

Reply:
i mean the very 1st msg in your PR, https://github.com/nim-lang/Nim/pull/18052#issue-649541477
it should contain the phrase `fix #17456` (without quotes) so that github auto-closes and so that it cross-references properly

(relying on commit messages is less good for a few reasons)